### DISCUSSION TITLE
run HFS automatically at boot up on windows server.

### DISCUSSION THREAD

**Discussion by GeorgeGeorgeGe** [rejetto/hfs#343] [Q&A]
Hello,

Would you please explain how to run HFS automatically at boot up (without user login) on windows server.

Thanks in advance.

**Comment by rejetto** [comment]
what you are asking is called "service".
Long ago I wrote these instructions https://github.com/rejetto/hfs#on-windows
but it's not for "exe" version.
Exe version saves configuration in a different folder, as described at https://github.com/rejetto/hfs#where-is-it-stored
If you want to launch configure the exe as a service you need different instructions, but I didn't try.
https://www.google.com/search?q=install+exe+as+service

**Reply by GeorgeGeorgeGe** [reply]
Thanks a lot.
